rootfs/scheduler/__init__.py

```diff
@@ -1024,9 +1019,9 @@ def _create_rc(self, namespace, name, image, command, **kwargs):  # noqa
                 secret = self._get_secret('deis', 'objectstorage-keyfile').json()
                 self._create_secret(namespace, 'objectstorage-keyfile', secret['data'])
 
-            l["image"] = image
+            l["slug_url"] = image
```

Comment:
we need to remove the registry part for the the image or the slugrunner will fail with invalid path as this is basically the path of the slug to be downloaded.